src/hotspot/cpu/riscv/stubRoutines_riscv.cpp

```diff
@@ -55,3 +55,225 @@ address StubRoutines::riscv::_string_indexof_linear_ul = nullptr;
 address StubRoutines::riscv::_large_byte_array_inflate = nullptr;
 
 bool StubRoutines::riscv::_completed = false;
+
+/**
+ *  crc_table[] from jdk/src/share/native/java/util/zip/zlib-1.2.5/crc32.h
```

Comment:
I think the correct path should be: `jdk/src/java.base/share/native/libzip/zlib/crc32.h`

Reply:
Fixed, thanks!